src/main/java/org/spongepowered/api/item/inventory/property/GuiIds.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.item.inventory.property;
+
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+
+/**
+ * An enumeration of all possible {@link GuiId}s in vanilla minecraft.
+ */
+public final class GuiIds {
+
+    // SORTFIELDS:ON
+
+    public static final GuiId CHEST = DummyObjectProvider.createFor(GuiId.class, "CHEST"); // "minecraft:chest"
+    public static final GuiId FURNACE = DummyObjectProvider.createFor(GuiId.class, "FURNACE"); // "minecraft:furnace"
+    public static final GuiId DISPENSER = DummyObjectProvider.createFor(GuiId.class, "DISPENSER"); // "minecraft:dispenser"
+    public static final GuiId CRAFTING_TABLE = DummyObjectProvider.createFor(GuiId.class, "CRAFTING_TABLE"); // "minecraft:crafting_table"
+    public static final GuiId BREWING_STAND = DummyObjectProvider.createFor(GuiId.class, "BREWING_STAND"); // "minecraft:brewing_stand"
+    public static final GuiId HOPPER = DummyObjectProvider.createFor(GuiId.class, "HOPPER"); // "minecraft:hopper"
+    public static final GuiId BEACON = DummyObjectProvider.createFor(GuiId.class, "BEACON"); // "minecraft:beacon"
+    public static final GuiId ENCHANTING_TABLE = DummyObjectProvider.createFor(GuiId.class, "ENCHANTING_TABLE"); // "minecraft:enchanting_table"
+    public static final GuiId ANVIL = DummyObjectProvider.createFor(GuiId.class, "ANVIL"); // "minecraft:anvil"
+    public static final GuiId VILLAGER = DummyObjectProvider.createFor(GuiId.class, "VILLAGER"); // "minecraft:villager"
+    public static final GuiId HORSE = DummyObjectProvider.createFor(GuiId.class, "HORSE"); // "EntityHorse"
```

Comment:
Can't we make the id like `minecraft:horse` and map it internally to `EntityHorse`? It would be nice for consistency and including the plugin id that provides the `GuiId`.